src/commands/deploy.js

```diff
@@ -4,7 +4,17 @@ import { setupDeployKey } from "./deploy-key.js";
 import { mkdir, readdir, writeFile, stat } from "node:fs/promises";
 import { dirname } from "node:path";
 import { InvalidPathError, KnownError } from "../lib/errors.js";
+import {read} from "read";
+import {apiDeploy} from "../lib/api.js";
 
+/**
+ *
+ * @param args
+ * @param stdin
+ * @param stdout
+ * @param stderr
+ * @param {Config} config
+ */
 export default async function deploy(args, { stdin, stdout, stderr }, config) {
   let key = config.deployKey;
   if (key === undefined) {
```

Comment:
Did you encounter this problem or is this defensive?

Reply:
Just to be consistent with the deploy-key.js